compiler/optimizer/UseDefInfo.cpp

```diff
@@ -2551,43 +2546,43 @@ void TR_UseDefInfo::buildUseDefs(TR::Node *node, void *vanalysisInfo, TR::BitVec
       // load of a symbol defines only symbol itself
       //
       numDefNodes = 1;
-      if (!aux._defsForSymbol[symIndex].IsZero() &&
+      if (!aux._defsForSymbol[symIndex]->isEmpty() &&
           (!sym ||
           (!sym->isShadow() &&
           !sym->isMethod())))
          {
             {
-            *analysisInfo -= aux._defsForSymbol[symIndex];
+            *analysisInfo -= (*aux._defsForSymbol[symIndex]);
```

Comment:
the * binding here is a bit odd - I think it is derefing the result of the [], but can the parens make this clear ie *() or (*()) rather than (*...)

Reply:
These have all been corrected. A silly mistake in a mass replace.